### ISSUE TITLE
WordPress has deleted this plugin and our other optimization plugins

### ISSUE THREAD

**Issue by optimalisatie** [o10n-x/wordpress-cloudfront-page-cache#1]
Topic moved: https://github.com/o10n-x/wordpress-css-optimization/issues/4

**Comment by szepeviktor** [comment]
@optimalisatie Hello!

The way things go is: just obey WP.org team and remove your company link, use different namespace etc.

This is the only solution I see.

**Comment by Non-State** [comment]
Use the GitHub method and align your plugins with the GitHub updater instead of using the WordPress plugin advertising system. Contact OzTheGreat (WPArtisan) he has experienced problems too in the past and most of his plugins are not directly available through the plugin interface, the same is also true of many other plugins and themes. Your plugins are very good and were working fine! All this makes me wonder if WP (the company) can do anything nefarious that could damage or block your plugin from working? If they can't you should promote it as a top solution, keep it open-source and use the GitHub updater to distribute it.

**Comment by optimalisatie** [comment]
HI @szepeviktor

Thank you for responding! It is very much appreciated!

I've been following your work on https://github.com/szepeviktor/w3-total-cache-fixed

For perhaps comparable reasons that you started a fork of W3 Total Cache, I believe that it is important that a plugin is published as it was designed. It is in the interest of users who simply should have the right to choose to use a solution or not. 

It would harm the plugin (and users) if the namespace would need to be changed. The plugins couldn't be published if the namespace needed to be changed. The namespace is shared between plugins to achieve single plugin performance while the individual plugin concept enables to have full focus on each aspect of optimization: HTML optimization, CSS optimization, Javascript optimization, Service Worker optimization etc. Users will be able to submit feedback for the specific optimization type, enabling to improve the plugins and optimization technologies much faster.

It appears that the motive for demanding the namespace change was intended to stall the publication of the plugin, I do not see an other explanation. It has nothing to do with security or code quality. They used the argument to delay publication for multiple weeks.

**Comment by optimalisatie** [comment]
@Non-State

Thank you very much for your feedback and advise! We will investigate the option to convert the plugins for use by a Github updater.

I want to repeat that there are no political or ideological motives at play from our side. This issue is very bad for the purpose of the plugins: an optimization plugin is about results only. Not about the author or anything else.

I simply needed to respond to what appears to have been a malicious attempt to delete the Above The Fold Optimization plugin in 2016. As a white hat SEO specialist with a clean reputation, a anti-spam link on 100 quality support messages that had cost a ton of effort and time, caused serious harm and wasn't acceptable. It needed to be addressed and due to lack of communication or response from WordPress it has escalated to the current deletion of the plugins.

A lot is unclear for us, but we value your feedback very much. As the plugins are free, users can find their way to use them if they would have an interest in them. There is not much else that we can do to improve user experience.

**Comment by optimalisatie** [comment]
One of our partners has tried to appease the situation and it is now clear that the issue will not be resolved. 

WordPress claims that they have been attacked (by the statement that we were treated unkind) and on that basis will never restore the plugins.

We did not (intend to) attack anyone, we have simply provided honest and open communication for the benefit of our users. It was a requirement to provide quality service.

In regards to the origin of the conflict WordPress argues that their signature rule needs to be obeyed and that they were correct to remove the signature. While that may be so, they didn't post the correct link, they linked to a anti-spam text. So they made a mistake and did so 100 times.

Their attitude was also very wrong. They were angry and aggressive. A polite response to the moderator, stating that a simple signature was removed, was deleted. They blocked access for 8 days after their action so that the author couldn't respond. Why? That can't be well intended. They certainly weren't serving or protecting users, they were attacking the plugin author.

For a white hat SEO specialist with a clean record, they caused direct harm. That could not have been accepted. The author needed to respond, and his choice was talking about it publicly, a choice that enables to resolve the issue in an intelligent way, with 'words'. It enables to get a more balanced perspective for the interest of any involved party.

A signature is something polite. While WordPress may have wanted to enforce their signature rule, they could have warned the author in a respectful and kind way. They were aggressive, angry and unkind. The author did not deserve such treatment for free support and solutions that users were happy with.

WordPress has now confirmed that they deleted the plugins for speaking truth. They don't allow our sincere 'words' on their platform.

Deletion of valuable optimization plugins causes harm to people. WordPress didn't serve the interest of users, they intend to serve themselves in an attempt to protect their good name.
